server/src/internalClusterTest/java/org/elasticsearch/search/aggregations/bucket/terms/StringTermsIT.java

```diff
@@ -1389,16 +1388,20 @@ public void testScriptWithValueType() throws Exception {
             assertThat(terms.getBuckets().size(), equalTo(1));
         }
 
-        String invalidValueType = source.replaceAll("\"value_type\":\"n.*\"", "\"value_type\":\"foobar\"");
+        String invalidValueType = source.replaceAll("\"value_type\":\"n.*?\"", "\"value_type\":\"foobar\"");
```

Comment:
Regexes are hard.  This test has always produced invalid json here, but apparently in the old code path we failed on value type validation first.  Stream parsing I guess.